Source/WTF/wtf/text/AtomString.cpp

```diff
@@ -32,6 +32,9 @@
 
 namespace WTF {
 
+WTF_EXPORT_PRIVATE StaticAtomString nullAtomData { nullptr };
```

Comment:
These should be const: `WTF_EXPORT_PRIVATE const StaticAtomString nullAtomData { nullptr };`

Reply:
Looks good. Changed.